recipes/packages.rb

```diff
@@ -16,4 +16,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-collect_packages(node['osl-php']['packages']).each { |p| package p }
+node['osl-php']['packages'].each do |p|
+  package p
+end
```

Comment:
Ok, this presents a problem with how it's currently implemented. We will likely need to update all related roles, and cookbooks that use this before this gets merged. A perfect test is running the php suite in proj-phpbb or proj-facebook.

It sets: ``node.default['osl-php']['packages']['proj-phpbb'] = %w(foo bar)``

I think we did this originally so that we could not have to worry about overriding attributes all the time. But I think we can just move away from that.

My recommendation: Update all cookbooks that use this recipe to use this new method.